# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/forms/moz.build
# Commit: 6141de941e71
# Full Hash: 6141de941e717cdb5485e1aa1ca00ffcd74f8a81
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2021-01-30 19:41:15
# Regressor Bug: 1683748
# File Overlap Count: 3
# Description:
#   Bug 1683748 - Support Grid/Flex/Table/Column layout for the rendered legend of a fieldset.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101555
# ==============================================================================

diff -r 3cf752580f1b -r 6141de941e71 layout/forms/moz.build
--- a/layout/forms/moz.build	Fri Jan 29 19:20:24 2021 +0000
+++ b/layout/forms/moz.build	Sat Jan 30 13:47:10 2021 +0000
@@ -27,7 +27,6 @@
     "nsGfxButtonControlFrame.cpp",
     "nsHTMLButtonControlFrame.cpp",
     "nsImageControlFrame.cpp",
-    "nsLegendFrame.cpp",
     "nsListControlFrame.cpp",
     "nsMeterFrame.cpp",
     "nsNumberControlFrame.cpp",